packages/kit/src/exports/vite/dev/index.js

```diff
@@ -175,15 +175,20 @@ export async function dev(vite, vite_config, svelte_config) {
 							const styles = {};
 
 							for (const dep of deps) {
-								const parsed = new URL(dep.url, 'http://localhost/');
-								const query = parsed.searchParams;
+								const url = new URL(dep.url, 'http://localhost/');
+								const query = url.searchParams;
 
 								if (
 									isCSSRequest(dep.file) ||
 									(query.has('svelte') && query.get('type') === 'style')
 								) {
+									// setting `?inline` to load CSS modules as css string
+									query.set('inline', '');
+
 									try {
-										const mod = await loud_ssr_load_module(dep.url);
+										const mod = await loud_ssr_load_module(
+											`${url.pathname}${url.search}${url.hash}`
```

Comment:
I think there won't ever be a `url.hash` since the browser doesn't sentdit to the server?

Reply:
The `dep.url` is actually from Vite imports, not from the browser.
The `url.hash` is present if we import with hash e.g. `import 'style.css?query#hash'` (Vite would throw error if hash is used without query).
I thought some plugins may use it?

Reply:
I think it would be very rare for imports to contain `#` (that usually cause issues with Vite too). But I guess it wouldn't hurt to support it if we want too, for correctness.

Reply:
Actually Ben might be right too as the `url` comes from the module graph, which is created from server requests, so they don't have hashes.

Reply:
```suggestion
											url.pathname + url.search
```

Reply:
Just checked the the hash is actually present, it's also sent from `vite.ssrLoadModule` to the plugins.
Don't know if anyone uses it, can remove if needed?

Reply:
Yeah, I just checked this too, the hash is indeed preserved. Sorry for the runaround, the existing code is perfectly correct